app/assets/stylesheets/postings.scss

```diff
@@ -1,3 +1,43 @@
 // Place all the styles related to the Postings controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+
+table, th, td {
+  border: 3px solid black;
+  margin: 15px;
+  padding: 15px;
+  font-weight: bold;
+}
+
+td {
+  background-color: AntiqueWhite;
+}
+
+th {
+  background-color: #006699;
+  align: center;
+  font-size: 15px;
+}
+.description-column {
+  width: 30%;
+}
+
+/* unvisited link */
```

Comment:
@jetamartin Why is there so much commented out CSS here? Can we delete this code?